tests/library_test.py

```diff
@@ -36,8 +36,58 @@ def __init__(self, *args, **kwargs):
 
     @responses.activate
     def test_get_documents(self):
-        responses.add(responses.GET, library.LIBRARY_URL + library.QUERY_START + "water", json=self.library_query, status=200)
+        responses.add(
+            responses.GET,
+            library.LIBRARY_URL + library.QUERY_START + "water",
+            json=self.library_query,
+            status=200,
+        )
         query_result = library.get_documents("water")
         self.assertIsInstance(query_result, dict)
         self.assertEqual(query_result["pages"], 10)
         self.assertEqual(len(query_result["docs"]), 10)
+
+
+class StudyRoomTest(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        unittest.TestCase.__init__(self, *args, **kwargs)
+        with (SAMPLE_PATH / "hillman_study_room_mock_response.json").open() as f:
+            self.hillman_query = json.load(f)
+
+    @responses.activate
+    def test_hillman_total_reserved(self):
+        responses.add(
+            responses.GET,
+            library.STUDY_ROOMS_URL,
+            json=self.hillman_query,
+            status=200
+        )
```

Comment:
```suggestion
        responses.add(
            responses.GET,
            library.STUDY_ROOMS_URL,
            json=self.hillman_query,
            status=200,
        )
```
`black` wants to collapse this down into a single line, but I assume you intended to write the arguments on separate lines. If you add a trailing comma to the last argument, it tells `black` that you don't want the arguments collapsed into a single line.

Reply:
Yeah that was a small error on my part. I thought I might need an extra argument but I didn't.